### PR TITLE
Further enhancement

### PR DESCRIPTION
**Pull request by j-mracek**

### CODE REVIEW

File: dnf-testing.sh [hunk 1]
```diff
@@ -121,6 +122,7 @@ build()
     if [ ${#output[@]} -eq 3 ] && \
        [ "${output[0]}" = "Successfully" ] && 
        [ "${output[1]}" = "built" ]; then
+        echo "${output[2]}" > container_id.txt
```
Comment (mhatina) [inline dnf-testing.sh]: I would save this into /tmp/container_id.txt. Also if it is not needed afterwards, please, delete the file.

File: dnf-testing.sh [hunk 2]
```diff
@@ -131,17 +133,28 @@ build()
...
+    if [ -z "$IMAGE" ]; then
+        if [ -f container_id.txt ]; then
+            IMAGE=$(cat container_id.txt)
+        else
+            fatal "Missing image name. Use -c option to define IMAGE";
+        fi
+    fi
     [ ${#TESTS[@]} -eq 0 ] && TESTS=("${FEATURES[@]}")
     local failed=0
+    local failed_name='Failed test(s):'
```
Comment (mhatina) [inline dnf-testing.sh]: I think 'failed_test_name' would be more verbose.

**Comment by j-mracek** [comment]
@mhatina It is ready just for you :-)

**Comment by j-mracek** [comment]
@mhatina I merged fixups. Please can you manually push the commits to master. (No dnf-bot). Thanks a lot.

**Pull request merged**